### PR TITLE
Sc 7821 dss auto line

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Comment by gbalabasquer** [comment]
We will need to update this PR as soon as we get this other PR merged: https://github.com/makerdao/dss-auto-line/pull/8

**Comment by gbalabasquer** [comment]
This is ready to be reviewed. Can I get a couple of approvals, please?

File: libexec/base-deploy
Comment (brianmcmichael) [inline libexec/base-deploy:298]: This doesn't appear to have been added to the 1.2.1 mcd-changelog update. I've added a card to clubhouse.

**Review by brianmcmichael — APPROVED** [review]

**Comment by gbalabasquer** [comment]
I'm gonna proceed merging this, so we can move on adding the latest collaterals.

**Pull request merged**
